tests/entrypoints/openai/test_rerank.py

```diff
@@ -8,17 +8,17 @@
 from ...utils import RemoteOpenAIServer
 
 MODEL_NAME = "BAAI/bge-reranker-base"
+DTYPE = "half"
 
 
 @pytest.fixture(scope="module")
 def server():
-    args = ["--enforce-eager", "--max-model-len", "100"]
+    args = ["--enforce-eager", "--max-model-len", "100", "--dtype", DTYPE]
```

Comment:
is this an intentional change to not use bf16, or a result of testing on CPU/older CUDA devices?